pysph/sph/wc/shift.py

```diff
@@ -0,0 +1,226 @@
+from math import sqrt
+from compyle.api import declare
+from pysph.sph.equation import Equation
+from pysph.base.reduce_array import serial_reduce_array
+from pysph.solver.tools import Tool
+
+
+class SimpleShift(Equation):
+    """See the paper by R. Xu et al. (JCP 2009), equation(35)
+    """
+    def __init__(self, dest, sources, const=0.04):
+        self.beta = const
+        super(SimpleShift, self).__init__(dest, sources)
+
+    def py_initialize(self, dst, t, dt):
+        from math import sqrt
+        vmag = dst.u[:]*dst.u[:] + dst.v[:]*dst.v[:] + dst.w[:]*dst.w[:]
```

Comment:
Can't this be simply `vmag = dst.u**2 + dst.v **2 + dst.w**2`

Reply:
fixed, thanks!